packages/p2p/src/utils/orders.ts

```diff
@@ -336,26 +534,44 @@ export default class ExtendedOrderDetails {
         };
     }
 
+    /**
+     * Function that returns the user details of the order
+     *
+     * @returns {object} user details
+     */
     get my_user_details() {
         return this.is_my_ad ? this.order_details.advertiser_details : this.order_details.client_details;
     }
 
+    /**
+     * Function that returns the user details of the order
+     *
+     * @returns {object} user details
+     */
     get other_user_details() {
         return this.is_my_ad ? this.order_details.client_details : this.order_details.advertiser_details;
     }
 
+    /**
+     * Function that returns the order expiry time in milliseconds
+     *
+     * @returns {number} expiry time in milliseconds
+     */
     get order_expiry_milliseconds() {
         return convertToMillis(this.order_details.expiry_time);
     }
 
+    /**
+     * Function to get the purchase time of the order
+     *
+     * @returns {string} purchase time
+     */
     get order_purchase_datetime() {
         return getFormattedDateString(new Date(convertToMillis(this.order_details.created_time)), true);
     }
-
-    get purchase_time() {
-        return getFormattedDateString(new Date(convertToMillis(this.order_details.created_time)), true);
-    }
```

Comment:
oh no i deleted purchase order_purchase_time in this pr 😆 https://github.com/binary-com/deriv-app/pull/10069